src/transformers/pipelines/token_classification.py

```diff
@@ -216,7 +217,7 @@ def _forward(self, model_inputs):
         }
 
     def postprocess(self, model_outputs, aggregation_strategy=AggregationStrategy.NONE):
-        outputs = model_outputs["outputs"]
+        outputs = model_outputs["outputs"].numpy()
```

Comment:
I'm not as familiar as you with the pipeline but will the `model_outputs["outputs"]`  can still be on GPU ? 
Maybe all preprocessing is firstly casted to CPU but I would rather double check. 
 
Else should we use `model_outputs["outputs"].cpu().numpy()` instead ?

Reply:
Hi @Ierezell,

Just like this file does not ever mention `.to(device)` it shouldn't mention `.cpu()`  as it's not the task of the pipeline to encapsulate that logic. Before it was done this way meaning many pipelines wouldn't actually support some features.

The logic is now in the `Pipeline.forward` method.

Rougly:
`preprocess`: (generic python objects) -> Tensors
`_forward`: Tensors -> Tensors
`postprocess`: Tensors -> Generic python objects

`forward` encapsulate classic logic for inference (inference mode, GPU tensor moving and other if needed, like batching)

Reply:
Ok, then everything will be at the good place at the good moment so that's perfect! Thanks for the clarification.